lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -225,7 +229,7 @@ impl RunnableModule for X64ExecutionContext {
         14:      41 ff e3        jmpq    *%r11
         */
         #[repr(packed)]
-        struct Trampoline {
+        struct LocalTrampoline {
```

Comment:
This structure has been renamed to avoid naming clashing with `wasmer_runtime_core::typed_func::Trampoline`.